src/core/api/option_parsers.ts

```diff
@@ -90,6 +90,8 @@ export interface ITransportOptions {
   referenceDateTime? : number;
   /** Allows to synchronize the server's time with the client's. */
   serverSyncInfos? : IServerSyncInfos;
+  /** Allows to adapt the switching audio logic depending the environment */
```

Comment:
This not depending on the environment, it's the applications which decide.

Reply:
yep, the application decides depending on the environment :)

Reply:
Depending on whatever it wants :p

It could also be just a functional need: wanting that an audio track switch happen directly.